packages/lib-core/package.json

```diff
@@ -32,8 +33,10 @@
     "@rollup/plugin-typescript": "^8.3.0",
     "@types/lodash-es": "^4.17.5",
     "@types/react": "^17.0.37",
+    "@types/react-router-dom": "5.1.2",
```

Comment:
Any reason not to use a semver pattern, but a specific version? (Just asking, I'm OK with this change.)

Reply:
The router v6 is not backwards compatible. But good point about adding carret in this one.